migration-guide.md

```diff
@@ -0,0 +1,57 @@
+# Swift Migration Guide
+
+This guide should help you with migrating projects on previous version of Swift to newer versions.
+
+### Swift 2 → Swift 3
+
+**IMPORTANT:** Make granular commits after each step and substep in this guide. Make sure the commit messages follow the proper rules.
+
+1. Clone project and pull/checkout all branches ([script here](http://stackoverflow.com/a/21189710/1001803)).
+    - Clean up all branches and keep only `master` and `develop`
+    - Make sure `develop` is even or ahead of `master`
+2. From `develop` branch run `git checkout -b feature/swift3-migration`
+3. *(optional)* Clean project, remove unused files, update file structure.
+4. Update `Cartfile` to point to Swift 3 versions of dependencies  
+    - Update framework names if used in project
+        - `NStack` → `NStackSDK`
+        - `Serializable` → `Serpent`
+    - Remove existing Carthage folder and run `carthage update --platform ios` (change platform as needed)
+5. *(optional)* Update translations generator 
+    - Download from latest Swift 3 release [on Github](https://github.com/nodes-ios/nstack-translations-generator/releases), do NOT add to Xcode
+    - Remove any old translations generator and templates present in project
+6. Open `YourProject.xcodeproj` in Xcode 8 and run Swift migrator
+    - Go through suggested changes to double check
+7. Go to Build Phases and modify current run scripts
+    - Update translations script as per [README](https://github.com/nodes-ios/nstack-translations-generator)
+    - Add or update script checking for TODO & FIXME
```

Comment:
No, as far as I can see, we can't have gists on the organization. 

One idea would be to create a `gists` repo on the organization. They're actually gonna be easier to navigate through if they're in a repo compared to normal gists. 
Or we can just link to it in the [project template](https://github.com/nodes-ios/project-template/blob/078ad339da3085f905067736886f2781a34ef12c/TemplateProject/TemplateProject.xcodeproj/project.pbxproj#L235) (kind of ugly, though).
Or you can just add the gist on your account and link to it. 

I'm fine with any of those solutions.